dsc/tests/dsc_config_get.tests.ps1

```diff
@@ -37,15 +37,15 @@ Describe 'dsc config get tests' {
             `$schema: https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2024/04/config/document.json
             resources:
             - name: Echo
-              type: Test/Echo
+              type: test/echo
```

Comment:
Why change to lower case?

Reply:
this was to explicitly test case-insensitivity